### PR TITLE
Classify OCC public maps by gamemodes

### PR DESCRIPTION
**Pull request by TheRealPear**
I took the time to go through individual maps, gave it a gamemode tag, and fixed any XML formatting issues (such as tab being used instead of spaces for indents, etc.) I also gave all the DTE maps the proper game title ("Destroy the Everything").

Although the gamemode tags aren't required, I believe that it is useful for identifying the correct gamemode for certain maps. Additionally, servers, like Warzone, have a custom plugin that checks for these data to calculate players' performance and stats during a match.

In short, about 559 maps has been cleaned up.

### CODE REVIEW

**Comment by CrazyisCreeps** [comment]
Debate as old as time, but is it Destroy the Everything or Destroy the Environment?

**Comment by TheRealPear** [comment]
> Debate as old as time, but is it Destroy the Everything or Destroy the Environment?

Probably the former. Destroy the Environment wouldn’t fit the scoreboard without modding.

**Comment by CoWinkKeyDinkInc** [comment]
It's Destroy the Everything.

File: dtcm/standard/athirk/map.xml [hunk 1]
```diff
@@ -1,11 +1,12 @@
 <map proto="1.4.0">
 <name>Athirk</name>
 <version>1.2</version>
+<objective>Leak the enemy team's cores and destroy their monuments to win!</objective>
+<gamemode>dtc</gamemode>
```
Comment (CoWinkKeyDinkInc) [inline dtcm/standard/athirk/map.xml:5]:
```suggestion
<gamemode>dtc</gamemode>
<gamemode>dtm</gamemode>
```

File: dtcm/standard/halcyon/map.xml [hunk 2]
```diff
@@ -1,125 +1,124 @@
...
-   </spawns>
-   <filters>
-      <filter name="deny-fire-spread">
-         <deny>
+<name>Halcyon</name>
+<version>2.2.1</version>
+<objective>Destroy the enemy team's monuments!</objective>
+<gamemode>dtm</gamemode>
+<authors>
+    <author>funkystudios</author> <!-- funkystudios -->
```
Comment (CoWinkKeyDinkInc) [inline dtcm/standard/halcyon/map.xml:8]:
```suggestion
    <author uuid="8242e55e-cc6a-458a-b272-a2b34918045f"/> <!-- funkystudios aka kaimyr -->
```
Noticed this change was already fixed on Warzone!

File: dtcm/standard/justice_2/map.xml [hunk 3]
```diff
@@ -1,107 +1,107 @@
...
-		</apply>
-		<cylinder id="red-spawn" base="-1999.5,24,-457.5" radius="15" height="20"/>
-		<apply region="red-spawn" block-place="never" block-break="never" enter="only-red"/>
-		<apply region="red-spawn" enter="only-red" message="You may not enter Red's base!"/>
-		<cylinder id="yellow-spawn" base="-1999.5,24,-191.5" radius="15" height="20"/>
-		<apply region="yellow-spawn" block-place="never" block-break="never" enter="only-yellow"/>
-		<apply region="yellow-spawn" enter="only-yellow" message="You may not enter Yellow's base!"/>
-	</regions>
-	<timelock>on</timelock>
+<map proto="1.4.2">
```
Comment (CoWinkKeyDinkInc) [inline dtcm/standard/justice_2/map.xml:2]: This XML still needs to be formatted properly. 4 spaces for an indent.

File: dtcm/standard/rendezvous_redux/map.xml [hunk 4]
```diff
@@ -1,125 +1,123 @@
...
-         <potion duration="5" amplifier="5">heal</potion>
-      </kit>
-      <kit name="red" parents="spawn">
+<name>Rendezvous Redux</name>
+<version>2.2.2</version>
+<objective>Destroy the enemy team's monuments!</objective>
+<gamemode>dtm</gamemode>
+<authors>
+    <author>Haribo98</author> <!-- Haribo98 -->
+    <author>Pavlo</author> <!-- Pavlo -->
```
Comment (CoWinkKeyDinkInc) [inline dtcm/standard/rendezvous_redux/map.xml:9]:
```suggestion
    <author uuid="378952b5-808f-4a33-911e-39c7ba153b38"/> <!-- Pavlo -->
```
Not sure which of these users is Haribo98 https://namemc.com/search?q=haribo98

File: dtcm/standard/sunrise_over_paradise/map.xml [hunk 5]
```diff
@@ -5,6 +5,7 @@
 <phase>development</phase>
 <genre>Objectives</genre>
```
Comment (CoWinkKeyDinkInc) [inline dtcm/standard/sunrise_over_paradise/map.xml:6]:
```suggestion

```

File: dtcm/standard/sunrise_over_sunrise_over_paradise/map.xml [hunk 6]
```diff
@@ -4,6 +4,7 @@
 <version>1.1.1</version>
 <genre>Objectives</genre>
```
Comment (CoWinkKeyDinkInc) [inline dtcm/standard/sunrise_over_sunrise_over_paradise/map.xml:5]:
```suggestion

```

File: dtcm/standard/tatakai/map.xml [hunk 7]
```diff
@@ -2,13 +2,12 @@
 <name>Tatakai</name>
 <version>1.0.0</version>
 <objective>Leak the enemy team's core to win</objective>
```
Comment (CoWinkKeyDinkInc) [inline dtcm/standard/tatakai/map.xml:4]:
```suggestion
<objective>Leak the enemy team's core to win!</objective>
```

File: dtcm/standard/ternio/map.xml [hunk 8]
```diff
@@ -1,123 +1,124 @@
...
-   <filters>
-      <filter name="deny-fire-spread">
-         <deny>
+<name>Ternio</name>
+<version>2.2.2</version>
+<phase>development</phase>
+<objective>Destroy the enemy team's monuments!</objective>
+<gamemode>dtm</gamemode>
+<authors>
+    <author>ParaPenguin</author>
```
Comment (CoWinkKeyDinkInc) [inline dtcm/standard/ternio/map.xml:9]:
```suggestion
    <author uuid="44df6b20-168f-4389-b7ac-434518207c2f"/> <!-- ParaPenguin -->
```

File: ffa/halloween/the_lobby_halloween/map.xml [hunk 9]
```diff
@@ -3,6 +3,7 @@
 <name>The Lobby: Halloween</name>
 <version>1.0.0</version>
 <phase>production</phase>
+<gamemode>ffa</gamemode>
 <objective>Get the most kills after 5 minutes!</objective>
 <authors>
     <author contribution="Lobby Builder">bleangamer</author>
```
Comment (CoWinkKeyDinkInc) [inline ffa/halloween/the_lobby_halloween/map.xml:9]:
```suggestion
    <author uuid="19388047-f04b-43ac-9546-22aa69aea75c" contribution="Lobby Builder"/> <!-- Bleanny/bleangamer -->
```

File: ffa/standard/cerulean_cave/map.xml [hunk 10]
```diff
@@ -2,6 +2,7 @@
 <name>Cerulean Cave</name>
 <version>1.0.3</version>
 <objective>Kill as many players as you can with your gun!</objective>
+<gamemode>ffa</gamemode>
 <authors>
     <author uuid="a23a1d91-7e97-40b4-8f5e-1ab82d8f5ade"/>
```
Comment (CoWinkKeyDinkInc) [inline ffa/standard/cerulean_cave/map.xml:7]:
```suggestion
    <author uuid="a23a1d91-7e97-40b4-8f5e-1ab82d8f5ade"/> <!-- chle_ -->
```

File: ffa/standard/cerulean_cave/map.xml [hunk 11]
```diff
@@ -2,6 +2,7 @@
 <name>Cerulean Cave</name>
 <version>1.0.3</version>
 <objective>Kill as many players as you can with your gun!</objective>
+<gamemode>ffa</gamemode>
 <authors>
     <author uuid="a23a1d91-7e97-40b4-8f5e-1ab82d8f5ade"/>
     <author uuid="dd495aa8-26a3-4b77-b3fa-e96e2e5a0d47"/>
```
Comment (CoWinkKeyDinkInc) [inline ffa/standard/cerulean_cave/map.xml:8]:
```suggestion
    <author uuid="dd495aa8-26a3-4b77-b3fa-e96e2e5a0d47"/>  <!-- McFanta_ -->
```

File: tdm/standard/rage_quit_tdm/map.xml [hunk 12]
```diff
@@ -3,6 +3,7 @@
 <name>Rage Quit TDM</name>
 <version>1.0.1</version>
 <objective>Eliminate the other team before 5 minutes is up</objective>
```
Comment (CoWinkKeyDinkInc) [inline tdm/standard/rage_quit_tdm/map.xml:5]:
```suggestion
<objective>Eliminate the other team before 5 minutes is up.</objective>
```

File: koth/standard/the_hill/map.xml [hunk 13]
```diff
@@ -4,6 +4,7 @@
 <version>1.1.6</version>
 <phase>development</phase>
 <objective>Capture and hold the hills</objective>
```
Comment (CoWinkKeyDinkInc) [inline koth/standard/the_hill/map.xml:6]:
```suggestion
<objective>Capture and hold the hills.</objective>
```
